Core/Core/Grades/View/GradeListView.swift

```diff
@@ -185,64 +202,98 @@ public struct GradeListView: View, ScreenViewTrackable {
+                    .resizable()
+                    .frame(width: 40, height: 40)
             }
+            .padding(.top, 12)
+            .padding(.horizontal, 16)
+            .padding(.bottom, 16)
         }
-        .padding(.top, 12)
-        .padding(.horizontal, 16)
-        .padding(.bottom, 16)
+    }
+
+    @ViewBuilder
+    private func courseLabelText() -> some View {
+        Text("Course", bundle: .core)
+            .foregroundStyle(Color.textDark)
+            .font(.regular14)
+            .accessibilityHidden(true)
+    }
+
+    @ViewBuilder
+    private func totalLabelText() -> some View {
+        Text("Total", bundle: .core)
+            .foregroundStyle(Color.textDark)
+            .font(.regular14)
+            .accessibilityHidden(true)
+    }
+
+    @ViewBuilder
+    private func totalGradeText(_ totalGrade: String) -> some View {
+        Text(totalGrade)
+            .foregroundStyle(Color.textDarkest)
+            .font(.semibold28)
+            .accessibilityLabel(Text("Total grade is \(totalGrade)", bundle: .core))
+            .accessibilityIdentifier("CourseTotalGrade")
+    }
+
+    @ViewBuilder
+    private func courseNameText(_ courseName: String) -> some View {
+        Text(courseName)
+            .foregroundStyle(Color.textDarkest)
+            .font(.semibold28)
+            .accessibilityLabel(Text("\(courseName) course"))
     }
 
     @ViewBuilder
     private func togglesView() -> some View {
         VStack(spacing: 0) {
-            Toggle(isOn: $isBaseOnGradedToggleOn) {
+            Toggle(isOn: $viewModel.baseOnGradedAssignment) {
                 Text("Base on graded assignments", bundle: .core)
                     .foregroundStyle(Color.textDarkest)
                     .font(.regular16)
-                    .fixedSize()
-                    .lineLimit(1)
             }
             .toggleStyle(SwitchToggleStyle(tint: Color(Brand.shared.primary)))
             .frame(height: 51)
             .padding(.horizontal, 16)
 
-            Divider()
-
-            Toggle(isOn: $isBaseOnGradedToggleOn) {
-                Text("Show What-if Score", bundle: .core)
-                    .foregroundStyle(Color.textDarkest)
-                    .font(.regular16)
-                    .fixedSize()
-                    .lineLimit(1)
-            }
-            .toggleStyle(SwitchToggleStyle(tint: Color(Brand.shared.primary)))
-            .frame(height: 51)
-            .padding(.horizontal, 16)
+//            Divider()
+//
+//            Toggle(isOn: $viewModel.isWhatIfScoreOn) {
+//                Text("Show What-if Score", bundle: .core)
+//                    .foregroundStyle(Color.textDarkest)
+//                    .font(.regular16)
+//                    .fixedSize()
+//                    .lineLimit(1)
+//            }
+//            .toggleStyle(SwitchToggleStyle(tint: Color(Brand.shared.primary)))
+//            .frame(height: 51)
+//            .padding(.horizontal, 16)
```

Comment:
Just to doublecheck: is the commented out part intentional?

Reply:
Yes, that's the What-if toggle that is not functional yet.